tests/MongoDB.Driver.Tests/AggregateGraphLookupTests.cs

```diff
@@ -0,0 +1,247 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System.Collections.Generic;
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Driver.Core.Misc;
+using MongoDB.Driver.Core.TestHelpers.XunitExtensions;
+using Xunit;
+
+namespace MongoDB.Driver.Tests
+{
+    public class AggregateGraphLookupTests
+    {
+        // private fields
+        private readonly IMongoDatabase _database;
+
+        // constructors
+        public AggregateGraphLookupTests()
+        {
+            var client = DriverTestConfiguration.Client;
+            var databaseName = CoreTestConfiguration.DatabaseNamespace.DatabaseName;
+            _database = client.GetDatabase(databaseName);
+        }
```

Comment:
Test setup in a constructor is no longer considered good practice. Instead have a helper method and call it from each test:

    var database = GetDatabase();

Reply:
Where `GetDatabase` is:

    private IMongoDatabase GetDatabase()
    {
        var client = DriverTestConfiguration.Client;
        var databaseName = CoreTestConfiguration.DatabaseNamespace.DatabaseName;
        return client.GetDatabase(databaseName);
    }

Reply:
Some reasons why test setup in a constructor is no longer considered good practice:

1. It's hidden code that is executed as a side effect and not visible in the test itself
2. It's not always the case that ALL tests in the file require the SAME setup